src/components/Sum.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Text = ({ a, b }) => (
```

Comment:
Component should also be named `Sum`

```suggestion
const Sum = ({ a, b }) => (
```